conf/ocsci/aws_config.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
We will need to add those variables to the default config here: default_config.yaml, cause currently we by default want to run on this AWS IPI without need to pass any config.

Reply:
Also can you please rename this file to aws_ipi_config.yaml or just aws_ipi.yaml

Reply:
Done